rfc-113-expanding-draft-access-for-unauthenticated-users.md

```diff
@@ -199,7 +215,26 @@ This was rejected since it was a significant departure from the existing access
 allowed to a fact checker through an auth-bypass token and may require
 considerable communication and consultation time. It remains a viable option
 but should be considered as part of a consideration of whether there should be
-more accountability in draft access.
+more accountability and/or simplicity in draft access.
+
+#### Determining which assets a page has to set auth-bypass ids on them
+
+An ideal scenario for asset access would be for it the mirror the system used
```

Comment:
I think there's a missing word here "would be for it the mirror the"

Reply:
d'oh - thanks sorted that out.